library/src/main/java/com/bumptech/glide/load/resource/bitmap/Downsampler.java

```diff
@@ -40,8 +40,8 @@
       "com.bumptech.glide.load.resource.bitmap.Downsampler.DecodeFormat", DecodeFormat.DEFAULT);
   /**
    * Indicates the {@link com.bumptech.glide.load.resource.bitmap.DownsampleStrategy} option that
-   * will be used to calculate the sample size to use to downsample an image given the original
-   * and target dimensions of the image.
+   * will be used to calculate the sample size to use to downsample an image given the original and
```

Comment:
Can you please revert irrelevant changes like comment/code formatting? It's harder to see what happened with all this diff. You could also squash your PR branch.